database/nakadi/10_data/04_tables/01_event_type.sql

```diff
@@ -2,6 +2,7 @@ CREATE TABLE IF NOT EXISTS zn_data.event_type (
   et_name varchar(255) NOT NULL PRIMARY KEY CHECK (et_name <> ''),
```

Comment:
I think as we don't really remove event types now we should remove this `PRIMARY KEY` because now it will be possible to have several event types with the same name when user removes event type and then creates another one with the same name.
And we should control the uniqueness of event type names only for event types with et_deleted = FALSE